### PR TITLE
NATIVE-458: Upgrade packages

### PR DESCRIPTION
**Pull request by klinzo**
Not upgraded:
- eslint configs (leads to many linting errors which I partly don't deem reasonable)
- react-navigation (major upgrade, issue created)
- react-navigation-stack (major upgrade, issue created)
- jest (no flow-types available for v25 yet)

### CODE REVIEW

File: src/modules/i18n/components/I18nProvider.js [hunk 1]
```diff
@@ -26,26 +26,13 @@ type StateType = {|
...
   appSettings: AppSettings
 
-  constructor () {
-    super()
+  constructor (props: PropsType) {
+    super(props)
 
     this.state = { errorMessage: null, initialisationFinished: false }
 
-    const i18nextResources = I18nProvider.transformResources(localesResources)
```
[the remark this answers is not in the document]
Comment (klinzo) [reply src/modules/i18n/components/I18nProvider.js]: Not removed but just moved, have a look at the `initI18n` method below.

File: src/modules/common/components/RemoteContent.js [hunk 2]
```diff
@@ -73,25 +72,24 @@ class RemoteContent extends React.Component<PropType, StateType> {
...
     const dataDetectorTypes: DataDetectorTypes = 'all'
     return <StyledView onLayout={this.onLayout}>
-      {// $FlowFixMe dataDetectorTypes (correct types, but Flow doesn't try the right branch)
-        <WebView
-          source={createHtmlSource(renderHtml(content, files, theme, RTL_LANGUAGES.includes(language) ? 'rtl' : 'ltr'),
-            URL_PREFIX + getResourceCacheFilesDirPath(cityCode))}
-          allowFileAccess // Needed by android to access file:// urls
-          originWhitelist={['*']} // Needed by iOS to load the initial html
-          javaScriptEnabled
+      <WebView
```
Comment (Schedulaar) [inline src/modules/common/components/RemoteContent.js]: The WebView has some breaking changes with the props (I upgraded it in NATIVE-9)

File: src/modules/i18n/components/I18nProvider.js [hunk 3]
```diff
@@ -26,26 +26,13 @@ type StateType = {|
 |}
 
 class I18nProvider extends React.Component<PropsType, StateType> {
-  i18n: i18n
```
Comment (Schedulaar) [inline src/modules/i18n/components/I18nProvider.js]: Would be nice to have this variable instead of working on the global i18n object

**Comment by maxammann** [comment]
What is the state of this PR? I think this should be merged soon.

**Comment by maxammann** [comment]
The WebView should probably be downgraded again, right?

**Comment by klinzo** [comment]
Yes and yes :)

**Pull request merged**